REFERENCES.md

```diff
@@ -0,0 +1,56 @@
+
+
+## Table of content
+
+
```

Comment:
1 unnecessary blank line

Reply:
@HeytemBou you should check the markdown formatting standards and/or add a markdown linter extension in your IDE :)